canonical/microk8s.io
# Apply /features redesign

## Done

- Apply redesign as per [doc](https://docs.google.com/document/d/1G6FijpahZCSfxcuFAkv22wKCILEkF2u4Q0TQUfkW5i4/edit) and [design](https://www.figma.com/file/0DTIS3uNNUuDAdCKWrk8Uf/23.10---Microk8s.io?node-id=94%3A4206&mode=dev)

## QA

- View the site locally in your web browser at: https://microk8s-io-613.demos.haus/features
- Check copy and design


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4184
